src/AnnouncementBubble/QuotedAnnouncementBubble.tsx

```diff
@@ -34,11 +35,26 @@ export const QuotedAnnouncementBubble: React.FC<Props> = ({
 }: Props) => {
   const [isExpanded, setisExpanded] = useState(false);
 
-  // If the content is not expanded, limit the message length so that screen readers
-  // don't attempt to read the entire hidden message. This only works for plain-string children
+  /* If the content is not expanded, limit the message length so that screen readers
+    don't attempt to read the entire hidden message. This only works for plain-string children */
   let content = children;
   if (!isExpanded && typeof content === "string") {
-    content = content.substring(0, 300);
+    let previewLength = 100;
```

Comment:
I ended up reverting this for now since 300 was adding a lot more text than what was actually visible in most cases. Also as far as I can tell, in most of our implementations the line break is just shy of 100 characters for both teachers and students

Reply:
Sounds good. My initial math was right then :D The only issue was in dewey's `AnnouncementBubbleView` which is irrelevant...